src/main/java/org/jboss/netty/handler/ssl/SslHandler.java

```diff
@@ -1656,8 +1656,7 @@ public void run() {
 
                 if (futures != null) {
                     final ClosedChannelException cause = new ClosedChannelException();
-                    final int size = futures.size();
-                    for (int i = 0; i < size; i ++) {
+                    for (int i = 0; i < futures.size(); i++) {
```

Comment:
Better to assign futures.size() to a variable first so it's not calculated every iteration.